### PR TITLE
README.md: Linuxbrew has been merged into Homebrew

### PR DESCRIPTION
**Pull request by sjackman**
After merging this PR, I'll tag Linuxbrew 1.9.4, which will automatically migrate users to Homebrew.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
 
+# Linuxbrew has been merged into Homebrew
+
+Linuxbrew/brew has been merged into [Homebrew/brew](https://github.com/Homebrew/brew)! Existing installations of Linuxbrew will be automatically migrated to Homebrew. Linuxbrew/brew will no longer be updated. See the [Homebrew documentation of Linuxbrew](https://docs.brew.sh/Linuxbrew) and the Homebrew 2.0.0 [blog post](https://brew.sh/2019/02/02/homebrew-2.0.0/).
+
+> Homebrew officially supports Linux and Windows 10 with Windows Subsystem for Linux (WSL). “Homebrew on Linux” is called “Linuxbrew”. You can install it in your home directory, so it does not require sudo, and use it to install software that your host distribution’s package manager does not provide. Linuxbrew uses its own repository for formulae: [Linuxbrew/homebrew-core](https://github.com/Linuxbrew/homebrew-core).
```
Comment (maxim-belkin) [inline README.md:7]: > does not require sudo

sudo is jargon. I suggest: "does not require administrative privileges (sudo)"

**Review by maxim-belkin — APPROVED** [review]
sigh

File: README.md (hunk 1 above)
Comment (sjackman) [reply README.md:7]: That wording is a direct quote from the Homebrew 2.0.0 blog post. Although I don't disagree, I won't edit it here.

**Comment by iMichka** [comment]
So the plan is to merge this PR, to tag 1.9.4, then merge #914, and then archive the repository?

**Comment by sjackman** [comment]
My preference is to retain the contents of the README.md for posterity. The section added by this PR and archiving the repo will make it clear that this repo is defunct.

File: README.md (hunk 1 above)
Comment (maxim-belkin) [reply README.md:7]: ah! that's a quote! all good then

**Comment by iMichka** [comment]
I am more worried of broken links, or wrong information that may be outdated at one point. I already stumbled on projects where they changed 3 times their repo in 10 years, and it was sometimes hard to follow where the correct information was. Nobody will come back to fix the readme later on.

**Comment by sjackman** [comment]
CI on Linux of PR #914 is failing. If we do decide to merge it, I'd like it to be 💚, in case we need to push a bug fix to this repo.
